src/Tizen.NUI.Components/Controls/Navigation/Page.cs

```diff
@@ -40,166 +39,37 @@ public class PageDisappearingEventArgs : EventArgs
     /// The Page class is a class which is an element of navigation.
     /// </summary>
     [EditorBrowsable(EditorBrowsableState.Never)]
-    public class Page : Control
+    public abstract class Page : Control
```

Comment:
could you explain why this is changed to `abstract` class?

Reply:
Because I think ContentPage and DialogPage are used for basic usages.

There would be more usages related to Page. So maybe more Page classes would be added later.

To give clear usage related to Page classes, I changed Page from class to abstract class.
ContentPage can be used for the full screen formatted Page cases (Title with Content layout).

In Xamarin forms, Page is not abstract class but it is hard to find Page class is used instead of other specific Page classes.
https://docs.microsoft.com/ko-kr/dotnet/api/xamarin.forms.page?view=xamarin-forms
https://docs.microsoft.com/ko-kr/xamarin/xamarin-forms/user-interface/controls/pages

Reply:
OK, I see. thank you for explanation.